src/main/java/org/dependencytrack/resources/v1/TeamResource.java

```diff
@@ -220,26 +192,77 @@ public Response deleteTeam(Team jsonTeam) {
         }
     }
 
+    @GET
+    @Path("/available-teams")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Returns a list of Teams what are available as selection", description = "Requires permission <strong>PORTFOLIO_MANAGEMENT</strong></p>")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "The Available Teams", content = @Content(schema = @Schema(implementation = AvailableTeams.class))),
+            @ApiResponse(responseCode = "401", description = "Unauthorized"),
+            @ApiResponse(responseCode = "404", description = "Teams could not be found")
+    })
+    @PermissionRequired(Permissions.Constants.PORTFOLIO_MANAGEMENT)
+    public Response availableTeams() {
+        UserPrincipal user;
+        boolean isAllTeams = false;
+        boolean required = false;
+        try (QueryManager qm = new QueryManager()) {
+            if (super.isLdapUser()) {
+                user = qm.getLdapUser(getPrincipal().getName());
+            } else if (super.isManagedUser()) {
+                user = qm.getManagedUser(getPrincipal().getName());
+            } else if (super.isOidcUser()) {
+                user = qm.getOidcUser(getPrincipal().getName());
+            } else {
+                return Response.status(401).build();
+            }
+            final List<Team> allTeams = qm.getTeams();
```

Comment:
Try to delay the loading of *all* teams until you know it's *really* required (i.e. ACL is disabled or user has the `ACCESS_MANAGEMENT` permission).